prototype/src/main/java/com/google/sps/data/Marker.java

```diff
@@ -0,0 +1,25 @@
+/** Represents a marker on the map. */
+public class Marker {
+
+	private final double lat;
```

Comment:
(Nitpick) Indents should be 2 spaces according to the [Java style guide](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)